stable_baselines3/common/policies.py

```diff
@@ -617,14 +660,19 @@ def evaluate_actions(self, obs: th.Tensor, actions: th.Tensor) -> Tuple[th.Tenso
         Evaluate actions according to the current policy,
         given the observations.
 
-        :param obs:
-        :param actions:
+        :param obs: Observation
+        :param actions: Actions
         :return: estimated value, log likelihood of taking those actions
             and entropy of the action distribution.
         """
         # Preprocess the observation if needed
         features = self.extract_features(obs)
-        latent_pi, latent_vf = self.mlp_extractor(features)
+        if not self.share_features_extractor:
```

Comment:
that's something I forgot but need to be mentioned in the doc: if the feature extractor is not shared, then you cannot share layers later on.

Reply:
Ok, I can add that to the doc!

Also, I could put a check after that line (670), something like "if shared_net not empty, raise error", but if, as you say in [this comment](https://github.com/DLR-RM/stable-baselines3/pull/1148#issuecomment-1299747968), you plan to remove the `shared_net` entirely, this might be useless.
Should I put the check in the meanwhile, and then remove it when the code gets refactored to remove the `shared_net`, in case you decide to do so?

Reply:
Ok I put it in the doc and I also put a check in the constructor of `ActorCriticPolicy`